code/controllers/Processes/supply.dm

```diff
@@ -226,7 +226,7 @@ var/datum/controller/supply/supply_controller = new()
 				A.req_access = list(SP.access)
 			else if(islist(SP.access))
 				var/list/L = SP.access // access var is a plain var, we need a list
-				A.req_access = L.Copy()
+				A.req_one_access = L.Copy()		//VOREStation Edit: Lets make sense
```

Comment:
Why?

Reply:
Basically, to allow existance of crates that can be opened via different accesses. The problem never came up because this section of code was unused up until now, with no supplypack in existance using more than a single access yet.

Reply:
Perhaps it would be better to go up to Polaris and add support for choice between a crate that requires one access, or requires all the accesses.

Reply:
I'll send that to Polaris, but until its implemented, I'll leave this PR using an edit.